### DISCUSSION TITLE
Ebai Estimator Error

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#462] [Questions]
I am running the ebai estimator and got a good plot for the fluxes/lc data but I am still getting an error message. This is probably because it is trying to run the ebai for the rv as well which is causing the error. I know the typically specification method b['lc'].plot()
but how do I specifically do this for the ebai because trying that normal method didn't work.
![image](https://user-images.githubusercontent.com/83960306/121400415-8e1a6700-c925-11eb-8550-ec49959701ba.png)

**Comment by kecnry** [comment]
I don't think the RV should be causing you any issues here (famous last words).  Can you paste the output of `print(b.filter(solution='ebai_sol'))`?  It looks to me from the error message that one of those is full of NaNs, for some reason (but then I'm not sure why the plot itself seems fine).  Does the error go away and the plot look fine if you pass `ylim=(0.5,1.1)`?

If you're willing to send me the bundle file, I can try to track down what's actually going on and fix the bug.  You can either attach it here if you're ok with it being shared publicly or email it to me.  Thanks!

**Reply by nbungart1999** [reply]
Here's the filter output:
![image](https://user-images.githubusercontent.com/83960306/121402603-169a0700-c928-11eb-9bb9-221b15a0646a.png)

**Reply by nbungart1999** [reply]
And the error did go away by setting that y limit! Why was that causing that error and how did that fix it?

**Reply by kecnry** [reply]
The lack of sigmas in your dataset was causing the issue.  Technically ebai itself doesn't require sigmas, but the plotting is passing them on as nans, and the automatic limits is attempting to make room for the errorbars, but is failing.  I'll open an issue to skip passing on the errorbars to plotting if not provided, and that should be a more permanent fix.  But until then, passing ylim will avoid the problem altogether.

**Reply by nbungart1999** [reply]
Ok, thanks! I also sent you the file, I don't have any other error messages just some minor warnings but nothing detrimental to the code.

**Reply by kecnry** [reply]
Ok, great!  I created issue #463 if you want to subscribe to that for updates.  It shouldn't be a difficult fix, but I might not be able to get to it right away.
